meetballs/urls.py

```diff
@@ -14,11 +14,12 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
-from meet_balls_app import views
+from django.urls import path, include
+from meet_balls_app import views as app_views
 
 
 urlpatterns = [
     path('admin/', admin.site.urls),
-    path('', views.home, name='home'),
+    path('', app_views.home, name='home'),
+    path('game-events/', include("game_event.urls"))
```

Comment:
May be better to add at the end of this line ','  to avoid problems later.
Should look like that this line:  path('game-events/', include("game_event.urls")),

Reply:
Sure